lua/lspsaga/init.lua

```diff
@@ -4,6 +4,7 @@ saga.saga_augroup = api.nvim_create_augroup('Lspsaga', { clear = true })
 
 local default_config = {
   ui = {
+    winbar_prefix = ' ',
```

Comment:
default value should be empty `''`

Reply:
I will amend the commit to set the default value to an empty string (''). Thank you for pointing that out! I forgot to change it.